### DISCUSSION TITLE
Code Overrides

### DISCUSSION THREAD

**Discussion by ORCACommander** [tgstation/tgstation-server-tools#1245] [Q&A] [answered]
ORCACommander — Today at 11:22 AM
in tgs 4, when using the code modifications folder, do i have to recreate the directory structure? Also can I just use a code snippet or do i have to use the whole file?

Cyberboss — Today at 12:04 PM
Yes, and it has to be the whole file unless you use HeadInclude.dm or TailInclude.dm

**Comment by Cyberboss** [comment, marked as answer]
Yes, and it has to be the whole file unless you use HeadInclude.dm or TailInclude.dm
